src/Reducers/books.js

```diff
@@ -0,0 +1,20 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../Actions';
+
+const initstate = { books: [{ Id: Math.random(), Title: 'Harry Potter & The Prisoners of Askaban', Category: 'Kids' }, { Id: Math.random(), Title: 'Shoe Dog', Category: 'Biography' }, { Id: Math.random(), Title: 'The Great Gatsby', Category: 'Action' }] };
```

Comment:
Let's move the `initstate to `src/index.js`

![image](https://user-images.githubusercontent.com/31897434/127383323-3e6e8ad2-a356-4485-8aae-ee74f4a74998.png)

Reply:
Creating initial state in the index.js causes cyclic dependency and throws linter errors as this state object has to be passed to the reducer